Packs/QRadar/Integrations/QRadar_v3/README.md

```diff
@@ -35,8 +35,13 @@ This integration was integrated and tested with API versions 10.1-14.0 on QRadar
 | Offenses (Read) | Offenses |
 | References (Create/Update) | Admin |
 | References (Read) | View Reference Data |
-## Mapping limitations for XSOAR users below 6.0.0
-* For XSOAR users below 6.0.0 version, using 'Pull from instance' option when creating a new mapper is not supported.
+## Mapping limitations for XSOAR versions below 6.0.0
+The *Pull from instance* option to create a new mapper is not supported in XSOAR versions below 6.0.0. 
+## Creating classifier using *Pull from instance* feature
+QRadar fetches incidents using a long-running execution. As a result, when you select the *Pull from instance* option to pull incidents from the QRadar service to create a classifier, it does not fetch offenses in real time, but uses samples to support the *Pull from instance* feature. This results in seeing the latest sample stored, and not the latest offense that was fetched.  
+## Important note regarding the *Query to fetch offenses* parameter
+The *Query to fetch offenses* feature enables you to define a specific query for offenses to be retrieved, e.g., **'status = OPEN and id = 5'**. The QRadar integration keeps track of IDs that have already been fetched in order to avoid duplicate fetching. 
+IIf you change the *Query to fetch offenses* value, it will not re-fetch offenses that have already been fetched. To re-fetch those offences, run the ***qradar-reset-last-run*** command. However, note that the list of QRadar IDs that had already been fetched will be reset and duplicate offenses could be re-fetched, depending on the user query.
```

Comment:
```suggestion
If you change the *Query to fetch offenses* value, it will not re-fetch offenses that have already been fetched. To re-fetch those offences, run the ***qradar-reset-last-run*** command. However, note that the list of QRadar IDs that had already been fetched will be reset and duplicate offenses could be re-fetched, depending on the user query.
```